app/src/main/java/com/fcaputo/parkingapp/mvp/model/SettingsModel.kt

```diff
@@ -0,0 +1,23 @@
+package com.fcaputo.parkingapp.mvp.model
+
+import com.fcaputo.parkingapp.mvp.contract.SettingsContract
+import com.fcaputo.parkingapp.utils.Constants
+import com.fcaputo.parkingapp.utils.validation.ValidationErrorType
+import com.fcaputo.parkingapp.utils.validation.ValidationResult
+
+class SettingsModel : SettingsContract.Model {
+    override fun validate(size: Int): ValidationResult {
+        return if (size == 0)
+            ValidationResult(isSuccess = false, ValidationErrorType.SIZE_IS_ZERO)
+        else
+            ValidationResult(isSuccess = true)
+    }
+
+    override fun save(newSize: Int) { parkingSize = newSize }
+
+    override fun getParkingSize(): Int = parkingSize
+
+    companion object {
+        var parkingSize: Int = Constants.SIZENOTSET_INT
```

Comment:
Why this is variable into a companion object? I think that is not needed

Reply:
I thought that maybe some other models would need to access this value in the future, so I decided to make it more like a "static" value

Reply:
But you have the `getParkingSize()` method in hand for that! 
Also, marking it as `var` you let that others may modify that as well

Reply:
Yes I agree with the "var" thing. So how could I expose the value for other models with that method? 
I must have an instance of this model in every other model that requires this value, right?